website/js/components/VariantSubmitter.js

```diff
@@ -0,0 +1,186 @@
+/*eslint-env browser */
+/*global require: false, module */
+'use strict';
+
+import React from "react";
+import {CollapsableMixin, Table} from "react-bootstrap";
+import classNames from 'classnames';
+import util from '../util';
+import KeyInline from './KeyInline';
+
+function sentenceCase(str) {
+    return str.replace(/\b\S/g, (t) => t.toUpperCase() );
+}
+
+// from https://www.ncbi.nlm.nih.gov/clinvar/docs/review_status/#revstat_def
+const marksToReviewStatuses = {
```

Comment:
did we ever get a consensus on whether or not stars will be included in the UI?

Reply:
I don't recall. Also, do you mean whether it should be a star glyph or not, or whether we should be including a rating at all?

Reply:
We should definitely include the rating, that much is clear.  In terms of whether or not to represent it as a star glyph, that seems less controversial than I thought.  ClinVar has SCV (report level) and RCV (variant level) review statuses, which they refer to with the same terms.  I'd been concerned that the stars in ClinVar were specific to the RCV data, and the RCV-level stars are readily-apparent ones, but having SCV (report) level stars doesn't seem misleading.

Reply:
sounds like we can keep it as is for now